app.js

```diff
@@ -1,8 +1,125 @@
 // import functions and grab DOM elements
+import { renderPastPoll } from "./render-utils.js";
+
+const optionForm = document.querySelector('#option-form');
+
+const optionAAddButton = document.querySelector('#add-option-a');
+const optionBAddButton = document.querySelector('#add-option-b');
+const optionAUndoButton = document.querySelector('#undo-option-a');
+const optionBUndoButton = document.querySelector('#undo-option-b');
+
+
+const optionAVotesEl = document.querySelector('#option-b-count');
+const optionBVotesEl = document.querySelector('#option-b-count');
+const optionATitleAEl = document.querySelector('#option-a-title');
+const optionBTitleEl = document.querySelector('#option-b-title');
+
+const questionEl = document.querySelector('#poll-question');
+const closePollEl = document.querySelector('#close-poll');
+
+const pastPollsEl = document.querySelector('#display-past-polls');
+
+
+
 
 // let state
+let question = '';
+let voteA = '';
+let voteB = '';
+let optionA = '';
+let optionB = '';
+
+let pastPollsArray = [];
+
 
 // set event listeners 
+
+optionForm.addEventListener('submit', (e) => {
+    e.prevenDefault();
+
+
+    const formData = new FormData(optionForm);
+
+    question = formData.get('question');
+    optionA = formData.get('optionA');
+    optionB = formData.get('optionB');
```

Comment:
Look at the names of the inputs in the form element in your HTML. Do they match what you are passing into `forData.get()`??